### PR TITLE
PBS Java vs Go differences doc update

### PR DESCRIPTION
**Pull request by RossGoncharuk**
Updated the document listing differences between Java and Go PBS versions

### CODE REVIEW

File: docs/differenceBetweenPBSGo-and-Java.md [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
 1) PBS-Java supports IP-address lookup in certain scenarios around GDPR. See https://github.com/rubicon-project/prebid-server-java/blob/master/docs/developers/PrebidServerJava_GDPR_Requirements.pdf
-1) PBS-Java supports InfluxDB and Graphite, PBS-Go supports InfluxDB and Prometheus as metrics backend.
+1) PBS-Java supports InfluxDB, Graphite and Prometheus, PBS-Go supports InfluxDB and Prometheus as metrics backend.
 1) PBS-Java has Circuit Breaker mechanism for database, http and geolocation requests. This can protect the server in scenarios where an external service becomes unavailable.
 1) PBS-Java supports `ext.prebid.cache.{bids,vastxml}.ttlseconds` field to pass the TTL to Cache Service 
-(per-impression, per-account and per-mediaType options are in progress [PR 154](https://github.com/rubicon-project/prebid-server-java/pull/154)),
+(per-impression option is in progress [PR 154](https://github.com/rubicon-project/prebid-server-java/pull/154)),
```
Comment (rpanchyk) [inline docs/differenceBetweenPBSGo-and-Java.md]: The `per-impression option` is completed also. See `org.prebid.server.cache.CacheService#cacheBidsOpenrtb` as ref.

File: docs/differenceBetweenPBSGo-and-Java.md [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
 ## Minor differences
 
 - PBS-Java removes null objects or empty strings (e.g. in Go `/auction` response bid object will have field `hb_cache: ""` whereas in Java it will be absent; also `digitrust: null` in PBS Go is not there in PBS Java). PBS-Go [Issue 476](https://github.com/prebid/prebid-server/issues/476)
-- Facebook AudienceNetwork adapter has been ported to use OpenRTB directly in PBS-Java. Other adapters will be ported in the near future. PBS-Go [Issue 211](https://github.com/prebid/prebid-server/issues/211)
+- All adapters have been ported to use OpenRTB directly in PBS-Java. PBS-Go [Issue 211](https://github.com/prebid/prebid-server/issues/211)
 - Java and Go adapter internal interface returns currency in different ways:
   - in PBS-Go, the adapter sets BidResponse.currency, which is outside of each TypedBid.
   - in PBS-Java, the adapter set BidderBid[N].currency.
-- PBS-Go fixed the IndexExchange-vs-IX issue. PBS-Java fix is in progress.
-- PBS-Java Rubicon adapter removes `req.cur` from request to XAPI.
```
Comment (rpanchyk) [inline docs/differenceBetweenPBSGo-and-Java.md]: Line required. Just change to `bidder`. See `org/prebid/server/bidder/rubicon/RubiconBidder.java:182` as ref.

File: docs/differenceBetweenPBSGo-and-Java.md [hunk 3]
```diff
@@ -1,36 +1,34 @@
...
  PBS-Go supports per-impression, per-responseBid and per-mediaType cache TTL options. Also PBS-Go use 60-seconds buffer to adjust cache TTL value.
 1) PBS-Java supports `ext.prebid.cache.{bids,vastxml}.returnCreative` field to control creative presence in response (`true` by default).
 1) PBS-Java supports checking the latest currency rates details, for example update time. This information is exposed via /currency-rates endpoint on admin port.
 
 ## Minor differences
 
 - PBS-Java removes null objects or empty strings (e.g. in Go `/auction` response bid object will have field `hb_cache: ""` whereas in Java it will be absent; also `digitrust: null` in PBS Go is not there in PBS Java). PBS-Go [Issue 476](https://github.com/prebid/prebid-server/issues/476)
-- Facebook AudienceNetwork adapter has been ported to use OpenRTB directly in PBS-Java. Other adapters will be ported in the near future. PBS-Go [Issue 211](https://github.com/prebid/prebid-server/issues/211)
+- All adapters have been ported to use OpenRTB directly in PBS-Java. PBS-Go [Issue 211](https://github.com/prebid/prebid-server/issues/211)
```
Comment (rpanchyk) [inline docs/differenceBetweenPBSGo-and-Java.md]: Pls specify `Facebook AudienceNetwork adapter` when mention `Issue 211`.

File: docs/differenceBetweenPBSGo-and-Java.md [hunk 4]
```diff
@@ -1,36 +1,34 @@
...
-October 26, 2018
+January 24, 2019
 
 The sister Prebid Server projects are both busy and moving forward at different paces on different features. Sometimes a feature may exist in one implementation
 and not the other for an interim period. This page tracks known differences that may persist for longer than a couple of weeks.
 
 ## Feature Differences
 
-1) PBS-Java supports Currency conversion. PBS-Go [issue 280](https://github.com/prebid/prebid-server/issues/280). PBS-Java [PR 22](https://github.com/rubicon-project/prebid-server-java/pull/22)
+1) PBS-Java supports Currency conversion. PBS-Go currently has it disabled as it's still under dev [issue 280](https://github.com/prebid/prebid-server/issues/280), [issue 760](https://github.com/prebid/prebid-server/pull/760). PBS-Java [PR 22](https://github.com/rubicon-project/prebid-server-java/pull/22)
```
Comment (rpanchyk) [inline docs/differenceBetweenPBSGo-and-Java.md]: Pls notice PBS-Go has Currency conversion implemented but disabled by default.

File: docs/differenceBetweenPBSGo-and-Java.md (hunk 1 above)
Comment (RossGoncharuk) [reply docs/differenceBetweenPBSGo-and-Java.md]: Then I guess it's not a difference anymore, other than "PBS-Go use 60-seconds buffer"

File: docs/differenceBetweenPBSGo-and-Java.md (hunk 2 above)
Comment (RossGoncharuk) [reply docs/differenceBetweenPBSGo-and-Java.md]: I guess I saw they've done the same at GO - https://github.com/prebid/prebid-server/pull/739

File: docs/differenceBetweenPBSGo-and-Java.md (hunk 4 above)
Comment (RossGoncharuk) [reply docs/differenceBetweenPBSGo-and-Java.md]: I've also added a note from their doc that it's due to this feature still being under dev.

File: docs/differenceBetweenPBSGo-and-Java.md (hunk 1 above)
Comment (rpanchyk) [reply docs/differenceBetweenPBSGo-and-Java.md]: yes

File: docs/differenceBetweenPBSGo-and-Java.md (hunk 2 above)
Comment (rpanchyk) [reply docs/differenceBetweenPBSGo-and-Java.md]: right! good catch.

File: docs/differenceBetweenPBSGo-and-Java.md (hunk 4 above)
Comment (rpanchyk) [reply docs/differenceBetweenPBSGo-and-Java.md]: ok, cool.

File: docs/differenceBetweenPBSGo-and-Java.md (hunk 1 above)
Comment (RossGoncharuk) [reply docs/differenceBetweenPBSGo-and-Java.md]: Actually, removed this one completely as this moment is already mentioned separately in Minor differences (60 sec buffer)

File: docs/differenceBetweenPBSGo-and-Java.md [hunk 5]
```diff
@@ -1,36 +1,33 @@
...
+1) PBS-Java supports InfluxDB, Graphite and Prometheus, PBS-Go supports InfluxDB and Prometheus as metrics backend.
 1) PBS-Java has Circuit Breaker mechanism for database, http and geolocation requests. This can protect the server in scenarios where an external service becomes unavailable.
-1) PBS-Java supports `ext.prebid.cache.{bids,vastxml}.ttlseconds` field to pass the TTL to Cache Service 
-(per-impression, per-account and per-mediaType options are in progress [PR 154](https://github.com/rubicon-project/prebid-server-java/pull/154)),
- PBS-Go supports per-impression, per-responseBid and per-mediaType cache TTL options. Also PBS-Go use 60-seconds buffer to adjust cache TTL value.
 1) PBS-Java supports `ext.prebid.cache.{bids,vastxml}.returnCreative` field to control creative presence in response (`true` by default).
 1) PBS-Java supports checking the latest currency rates details, for example update time. This information is exposed via /currency-rates endpoint on admin port.
+1) PBS-Java has a specific `/cookie-sync` behaviour, that updates `uids.HOST-BIDDER` from `host-cookie` if first is absent or not equal to second.
+1) PBS-Java sets `/setuid` in `usersync-url`.
```
Comment (rpanchyk) [inline docs/differenceBetweenPBSGo-and-Java.md]: Pls add more details on this, see #256 description.

**Pull request merged**